### PR TITLE
Updated Entity mappings of analytical rules

### PR DESCRIPTION
**Pull request by v-shukore**
Required items, please complete
   
   Change(s):
   - Solution repackaging

   Reason for Change(s):
   - PR #11119 

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

### CODE REVIEW

File: Solutions/Threat Intelligence/Package/createUiDefinition.json [hunk 1]
```diff
@@ -60,91 +60,7 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs five (5) data connectors for ingesting threat indicators (IP addresses, domains, URLs and file hashes) into Microsoft Sentinel. The ingested threat indicators can be used for correlation to enable monitoring, alerting, and hunting using your threat intelligence."
```
Comment (v-sabiraj) [inline Solutions/Threat Intelligence/Package/createUiDefinition.json:63]: @v-shukore , revert the changes for data connector description, thanks.

**Review by v-sabiraj — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]
